configure/recipes/default.rb

```diff
@@ -1,3 +1,14 @@
+node.default[:commit_hash] = "null"
+
+ruby_block "jbx_version" do
+  block do
+    %s(su #{node[:user]; git config --global --add safe.directory #{node["jbx"]["path"]})
+    commit_hash = %x(cd #{node["jbx"]["path"]}; git rev-parse HEAD)
+    node.default[:commit_hash] = "#{commit_hash.strip}"
```

Comment:
You want to use `run_state` here rather than `node.default`
https://docs.chef.io/recipes/#noderun_state

Reply:
Ah, ok. I'll go back and switch it out.
I spent a whack of time on options that may have worked but, I originally had the block in the wrong location where it wasn't actually passing the value onto further configurations.
I'll make this change.